packages/ember-glimmer/lib/environment.js

```diff
@@ -268,8 +268,9 @@ export default class Environment extends GlimmerEnvironment {
 
   getComponentDefinition(path, parentMeta) {
     let name = path[0];
+    let owner = !!(parentMeta && parentMeta.owner) ? parentMeta.owner : this.owner;
```

Comment:
Went to look into this more since I wrote this code, then noticed that `hasComponentDefinition` returns false in ember. Why is this?

https://github.com/emberjs/ember.js/blob/5f5b9ae5f8c117c3dc45409b7f0d2272f042cb68/packages/ember-glimmer/lib/environment.js#L266